src/cli/fuzzer_bridge/models.py

```diff
@@ -0,0 +1,140 @@
+
+    @field_validator("storage", mode="before")
+    @classmethod
+    def validate_storage(cls, v):
+        """Convert storage keys and values to HexNumber."""
+        if not v:
+            return {}
+        return {HexNumber(k): HexNumber(v_) for k, v_ in v.items()}
+
+
+class FuzzerAuthorization(BaseModel):
+    """Authorization tuple for EIP-7702 SetCode transactions."""
+
+    chainId: HexNumber = Field(..., alias="chainId")
+    address: Address
+    nonce: HexNumber
+    v: HexNumber  # yParity
+    r: HexNumber
+    s: HexNumber
+
+    class Config:
+        """Pydantic configuration."""
+
+        populate_by_name = True
+
+
+class FuzzerTransaction(BaseModel):
+    """Transaction definition in fuzzer output."""
+
+    from_: Address = Field(..., alias="from")
+    to: Optional[Address] = None
+    value: HexNumber = HexNumber(0)
+    gas: HexNumber
+    gasPrice: Optional[HexNumber] = Field(None, alias="gasPrice")
+    maxFeePerGas: Optional[HexNumber] = Field(None, alias="maxFeePerGas")
+    maxPriorityFeePerGas: Optional[HexNumber] = Field(None, alias="maxPriorityFeePerGas")
+    nonce: HexNumber
+    data: Bytes = Bytes(b"")
+    accessList: Optional[List[Dict[str, Union[str, List[str]]]]] = Field(None, alias="accessList")
+    blobVersionedHashes: Optional[List[Hash]] = Field(None, alias="blobVersionedHashes")
+    maxFeePerBlobGas: Optional[HexNumber] = Field(None, alias="maxFeePerBlobGas")
+    authorizationList: Optional[List[FuzzerAuthorization]] = Field(None, alias="authorizationList")
+
+    class Config:
+        """Pydantic configuration."""
+
+        populate_by_name = True
+
+
+class FuzzerEnvironment(BaseModel):
+    """Environment definition in fuzzer output."""
+
+    currentCoinbase: Address = Field(..., alias="currentCoinbase")
+    currentDifficulty: HexNumber = Field(HexNumber(0), alias="currentDifficulty")
+    currentGasLimit: HexNumber = Field(..., alias="currentGasLimit")
+    currentNumber: HexNumber = Field(..., alias="currentNumber")
+    currentTimestamp: HexNumber = Field(..., alias="currentTimestamp")
+    currentBaseFee: Optional[HexNumber] = Field(None, alias="currentBaseFee")
+    currentRandom: Optional[Hash] = Field(None, alias="currentRandom")
+    currentExcessBlobGas: Optional[HexNumber] = Field(None, alias="currentExcessBlobGas")
+    currentBlobGasUsed: Optional[HexNumber] = Field(None, alias="currentBlobGasUsed")
+    parentBeaconBlockRoot: Optional[Hash] = Field(None, alias="parentBeaconBlockRoot")
+    parentUncleHash: Optional[Hash] = Field(None, alias="parentUncleHash")
+    parentDifficulty: Optional[HexNumber] = Field(None, alias="parentDifficulty")
+    parentBaseFee: Optional[HexNumber] = Field(None, alias="parentBaseFee")
+    parentGasUsed: Optional[HexNumber] = Field(None, alias="parentGasUsed")
+    parentGasLimit: Optional[HexNumber] = Field(None, alias="parentGasLimit")
+
+    class Config:
+        """Pydantic configuration."""
+
+        populate_by_name = True
```

Comment:
This is basically `from ethereum_test_types import Environment`, so we should just simply reuse that.